### PR TITLE
release_v1.5.5

### PR DESCRIPTION
**Pull request by ShahriyarR**
Merge it at the end.

### CODE REVIEW

**Comment by ShahriyarR** [comment]
Can be merged as I have tested basic functionality works fine here.

File: setup.py [hunk 1]
```diff
@@ -8,21 +8,20 @@
...
     url='https://github.com/ShahriyarR/MySQL-AutoXtraBackup',
-    download_url='https://github.com/ShahriyarR/MySQL-AutoXtraBackup/archive/v1.5.4.zip',
+    download_url='https://github.com/ShahriyarR/MySQL-AutoXtraBackup/archive/v1.5.5.zip',
     license='MIT',
     author='Shahriyar Rzayev',
     author_email='rzayev.shahriyar@yandex.com',
     description='Commandline tool written in Python 3 for using Percona XtraBackup',
     install_requires=[
         'click>=3.3',
-        'mysql-connector-python==2.1.4',
```
Comment (BarbzYHOOL) [inline setup.py]: sure??

Comment (ShahriyarR) [reply setup.py]: Master said yes :)
https://github.com/ShahriyarR/MySQL-AutoXtraBackup/blob/51afdd437fd67bede9719d46c5d5b0c94a6304dc/setup.py#L26

Comment (BarbzYHOOL) [reply setup.py]: but where is the mysql connector library then? (sorry i'm blind xD)

Comment (ShahriyarR) [reply setup.py]: It turned out there is no need for this library :D ahaahah )

Comment (BarbzYHOOL) [reply setup.py]: o_0 ok then lol

**Comment by bimith** [comment]
Hi,
What am telling part will not match for above discussion  
About if the backup data is older which means 1 week or one month old this will be get deleted by automatically from directory or we need to do the deleting part.  

Kindly help me

**Pull request merged**